e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -504,106 +501,182 @@ describe("issues 20845, 25031", () => {
   });
 });
 
-describe.skip("27643", { tags: "@external" }, () => {
+// TODO:
+// - Add tests for embedding previews in both cases
+// - Add tests for disabled, editable and locked parameters in both cases
```

Comment:
I discussed this in the embedding team sync that we'll tackle this in [a new P2 issue](https://github.com/metabase/metabase/issues/45997), so we could focus on closing this P1 ASAP.

Reply:
I agree for previews, but there's nothing stopping you from testing the actual disabled, editable and locked params already in this PR.

However, if the follow up is planned soon - ignore this comment.